x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/support/DelegatedAuthorizationSupport.java

```diff
@@ -49,6 +50,13 @@
      */
     public DelegatedAuthorizationSupport(Iterable<? extends Realm> allRealms, RealmConfig config, XPackLicenseState licenseState) {
         this(allRealms, config.getSetting(AUTHZ_REALMS), config.settings(), config.threadContext(), licenseState);
+        if (lookup.hasRealms()) {
+            logger.info(
```

Comment:
As discussed, this is printed once during startup, so it's not going to be noisy. (If it were printed on each authentication, it would definitely need to be debug/trace)